config/jobs/image-pushing/k8s-staging-apisnoop.yaml

```diff
@@ -44,3 +44,26 @@ postsubmits:
               - --scratch-bucket=gs://k8s-staging-apisnoop-gcb
               - --env-passthrough=PULL_BASE_REF
               - apps/snoopdb
+  kubernetes-sigs/verify-conformance:
+    - name: apisnoop-push-verify-conformance
```

Comment:
this name is unusual which isn't strictly a problem but might cause some confusion later, usually it's like $repo-push-images or $repo-push-$target

Reply:
Good point! I've updated it in 66f1b27891e043fc5c257b6121b5eb0b659f0289